.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  run-build:
+    name: Build govuk-design-system
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read node version from .nvmrc
+      id: nvm
+      run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+
+    - name: Setup node
+      uses: actions/setup-node@v1
+      with:
+        node-version: "${{ steps.nvm.outputs.NVMRC }}"
+
+    - name: Install dependencies
+      run: npm install
```

Comment:
In `travis.yml` we do:

https://github.com/alphagov/govuk-design-system/blob/fbb970e8e6aa5771db864694227df56f07706ef6/.travis.yml#L27

Not sure why we include the `--no-optional` flag and it might make sense to remove it, but want to make sure it's a conscious decision.

Reply:
Thanks, that wasn't a conscious decision! It doesn't look like we explicitly set any optional dependencies ourselves, but some of our dependencies do so it probably makes sense to keep the flag.

Reply:
Updated: https://github.com/alphagov/govuk-design-system/pull/1436/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R23